tests/unit/modules/network/junos/test_junos_ospfv3.py

```diff
@@ -0,0 +1,301 @@
+                        ],
+                    )
+                ],
+                state="replaced",
+            )
+        )
+
+        result = self.execute_module(changed=True)
+        commands = result["commands"]
+        self.execute_module(changed=True, commands=commands)
+
+    def test_junos_ospfv3_overridden(self):
+        set_module_args(
+            dict(
+                config=[
+                    dict(
+                        router_id="10.200.16.75",
+                        areas=[
+                            dict(
+                                area_id="0.0.0.100",
+                                stub=dict(default_metric=200, set=True),
+                                interfaces=[
+                                    dict(
+                                        name="so-0/0/0.0", priority=3, metric=5
+                                    )
+                                ],
+                            )
+                        ],
+                    )
+                ],
+                state="overridden",
+            )
+        )
+        commands = [
+            '<nc:protocols xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0"><nc:ospf3 delete="delete"/><nc:ospf3>'
+            "<nc:area><nc:name>0.0.0.100</nc:name><nc:interface><nc:name>so-0/0/0.0</nc:name>"
+            "<nc:priority>3</nc:priority>"
+            "<nc:metric>5</nc:metric></nc:interface>"
+            "<nc:stub><nc:default-metric>200</nc:default-metric></nc:stub></nc:area></nc:ospf3></nc:protocols>",
+            '<nc:routing-options xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0">'
+            "<nc:router-id>10.200.16.75</nc:router-id></nc:routing-options>",
+        ]
+        result = self.execute_module(changed=True)
+        self.assertEqual(sorted(result["commands"]), commands)
+
+    def test_junos_ospfv3_overridden_idempotent(self):
+        self.get_connection.return_value = load_fixture(
+            "junos_ospfv3_config.cfg"
+        )
+        src = load_fixture("junos_ospfv3.cfg", content="str")
+        set_module_args(dict(src=src))
+        set_module_args(
+            dict(
+                config=[
+                    dict(
+                        router_id="30",
+                        areas=[
+                            dict(
+                                area_id="100",
+                                stub=dict(default_metric=10, set=True),
+                                interfaces=[
+                                    dict(
+                                        name="so-0/0/0.0", priority=3, metric=5
+                                    )
+                                ],
+                            )
+                        ],
+                    )
+                ],
+                state="overridden",
+            )
+        )
+
+        result = self.execute_module(changed=True)
+        commands = result["commands"]
+        self.execute_module(changed=True, commands=commands)
```

Comment:
Please add testcases for the rest of the states as well. This will increase the test coverage percentage.